### ISSUE TITLE
Full test coverage of message specifications

### ISSUE THREAD

**Issue by treiher** [Componolit/RecordFlux-specifications#29]
The test coverage for all message specifications should be 100 %. Missing test data for a message should lead to an error instead of a warning.

**Comment by senier** [comment]
@treiher Is this a duplicate of #10?

**Comment by treiher** [comment]
Not really. Both are similar, but while #10 is about adding some test data for all specifications (i.e. achieving a test coverage of > 0 %), this issue is about reaching 100 % test coverage for all specifications. While the former is achievable with limited effort, I'm not sure if we can accomplish the latter in the near term.

**Comment by senier** [comment]
I see, so #10 is about doing something and #29 is about doing everything. Then we should keep both to track the long-term effort.
